README.md

```diff
@@ -357,8 +357,9 @@ In the example below you can see how, and also what the default settings are.
 	      excludedTypes: [],
 	      excludedConfig: [
 	        "core-store.plugin_users-permissions_grant",
-          "core-store.plugin_upload_metrics",
-          "core-store.strapi_content_types_schema",
+          	"core-store.plugin_upload_metrics",
+          	"core-store.strapi_content_types_schema",
+		"core-store.ee_information",
```

Comment:
![image](https://user-images.githubusercontent.com/5942270/222127656-3e8c6652-c791-429a-be76-b98c695b7913.png)

no idea why github shows the alignment like this

Reply:
LGTM